src/World.h

```diff
@@ -757,6 +757,9 @@ class cWorld :
 		return (IsWeatherWet() && !IsBiomeNoDownfall(GetBiomeAt(a_BlockX, a_BlockZ)));
 	}
 
+	/** Returns the seed of the world. */
+	int GetSeed(void) { return m_Generator.GetSeed(); }
```

Comment:
I don't like the fact that this is in cWorld. The Seed is not cWorld's responsibility.